### DISCUSSION TITLE
Allow custom images when deploying multiple Ragapps

### DISCUSSION THREAD

**Discussion by Rauljauregi** [ragapp/ragapp#190] [Q&A]
I would like the ability to choose the image of the new Ragapp application when deploying with multiple Ragapps. This way, we can customize Ragapp, create our own image, and then deploy it as multiple Ragapps, taking advantage of the benefits that come with deploying multiple instances.

I have to say that I tried changing the image name inside docker-compose.yml from ragapp/ragapp to the customized one ragapp-custom , made using the dockefile that is in ragapp/src/ragapp. 

This image works for a single ragapp but not for multiple ragapps deployment method.

**Comment by marcusschiesser** [comment]
You can replace the images if you follow this description: https://github.com/ragapp/ragapp/blob/main/deployments/multiple-ragapps/README.md#use-the-test-image-main-branch
